langchain/chains/conversational_retrieval/base.py

```diff
@@ -161,11 +167,7 @@ def from_llm(
         **kwargs: Any,
     ) -> BaseConversationalRetrievalChain:
         """Load chain from LLM."""
-        doc_chain = load_qa_chain(
-            llm,
-            chain_type=chain_type,
-            prompt=qa_prompt,
-        )
+        doc_chain = load_qa_chain(llm, chain_type=chain_type, prompt=qa_prompt,)
```

Comment:
These lower changes aren't needed :)

Reply:
Got it ,Thanks
and 
I'm new to open-source but eager to contribute. Could you offer some guidance on where to start? I can dedicate around an hour each day. Thank you.

Best regards,
Kanukollu Goutham

Reply:
This PR is helpful! I extended your PR in #3236 to update the - we're working on some larger improvements, for this to improve message passing, but it's great to be helping on the issues as you have done

Reply:
Thank you for the kind words! I'm glad that my pull request was helpful, and it's great to hear that you could build on it to make further improvements. I'm always happy to contribute to open-source projects and help others, so please feel free to reach out if you have issues I can help with.